prime-router/src/main/kotlin/Sender.kt

```diff
@@ -223,12 +227,12 @@ class FullELRSender : Sender {
         primarySubmissionMethod
     )
 
-    constructor(copy: FullELRSender) : this(
+    constructor(copy: FullELRSender, shouldEmptyKeys: Boolean = false) : this(
```

Comment:
I opted to re-use the constructor with a default param, rather than make a new one.  This code will be removed once the migration is complete.